api/desecapi/management/commands/check-slaves.py

```diff
@@ -0,0 +1,80 @@
+from datetime import timedelta
+from socket import gethostbyname
+from time import sleep
+
+from django.conf import settings
+from django.core.mail import get_connection, mail_admins
+from django.core.management import BaseCommand
+from django.utils import timezone
+import dns.message, dns.query, dns.rdatatype
+
+from desecapi import pdns
+from desecapi.models import Domain
+
+
+def query_serial(zone, server):
+    query = dns.message.make_query(zone, 'SOA')
+    response = dns.query.tcp(query, server)
+
+    for rrset in response.answer:
```

Comment:
this loop can be replaced by `response.find_rrset(dns.message.ANSWER, zone, dns.rdataclass.IN, dns.rdatatype.SOA)`; it may be necessary to update zone using `dns.name.from_text`